src/components/ux-tooltip/ux-tooltip.feature

```diff
@@ -0,0 +1,8 @@
+Feature: ux-tooltip test suite
+
+	Scenario: Loading ux-tooltip
+		Given I have loaded component "ux-tooltip" with use case "dataDriven"
+		When I hover "link"
```

Comment:
From a grammatical perspective, we should probably change this step to read:

"When I hover the "FOO" element"

or similar.